src/components/shared/NodeCountSelector.js

```diff
@@ -57,7 +59,7 @@ class NodeCountSelector extends React.Component {
 
     if (this.props.autoscalingEnabled === true) {
       return (
-        <form onSubmit={this.handleFormSubmit}>
```

Comment:
This shouldn't be a `<form />` element. This is an input, it should be part of a `<form />` element together with other inputs. Using a `<form />` element inside another one is invalid HTML markup